### PR TITLE
Cardlize detail

### PR DESCRIPTION
**Pull request by henryzbw**
image

### CODE REVIEW

File: src/languageProvider/locales/ko-KR.json [hunk 1]
```diff
@@ -411,7 +411,7 @@
   "walletUnlockDialog.walletPassphrase": "지갑 패스프레이즈",
   "walletUnlockDialog.walletPassphraseRequired": "이 작업을 수행하려면이 지갑의 잠금을 해제해야합니다. 지갑 패스프레이즈를 입력하십시오.",
   "walletUnlockDialog.walletUnlocked": "지갑이 잠금 해제되었습니다.",
-  "withdrawDetail.alreadyWithdrawn": "본 주소로 이미 철회를 했습니다.",
+  "withdrawDetail.alreadyWithdrawn": "이미 철회했습니다.",
```
Comment (dwalintukan) [inline src/languageProvider/locales/ko-KR.json]: you speak korean?

File: src/models/BetOption.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import _ from 'lodash';
```
Comment (dwalintukan) [inline src/models/BetOption.js]: import only what you need

File: src/scenes/CreateEvent/store.js [hunk 3]
```diff
@@ -65,7 +65,7 @@ const MAX_LEN_EVENTNAME_HEX = 640;
 const MAX_LEN_RESULT_HEX = 64;
 const TIME_DELAY_FROM_NOW_SEC = 15 * 60;
 const TIME_GAP_MIN_SEC = 24 * 60 * 60;
-const VALIDATE_TIME_GAP_MIN_SEC = 30 * 60;
+const VALIDATE_TIME_GAP_MIN_SEC = 1 * 60;
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/store.js]: this intentional?

File: src/scenes/Event/Option/index.js [hunk 4]
```diff
@@ -76,23 +78,25 @@ export default class Option extends Component {
             expanded={isExpanded(selectedOptionIdx) || skipExpansion}
             onChange={skipExpansion ? null : () => eventPage.setSelectedOption(idx)}
             disabled={option.disabled || disabled}
+            classes={{ root: classes.expansionPanelRoot, disabled: classes.expansionPanelDisabled }}
           >
-            <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
+            <ExpansionPanelSummary expandIcon={isPrevResult ? <img src="/images/icon-tick_bordered.svg" alt='leaderboard' /> : <ExpandMoreIcon />} classes={{ root: classes.expansionPanelSummaryRoot, content: classes.expansionPanelSummaryContent, expandIcon: cx(classes.expandIcon, betSpecific && classes.hideIcon), disabled: classes.expansionPanelSummaryDisabled }}>
```
Comment (dwalintukan) [inline src/scenes/Event/Option/index.js]: nit: place values on separate lines for readability

File: src/scenes/Event/Option/styles.js [hunk 5]
```diff
@@ -1,21 +1,18 @@
...
-      paddingBottom: theme.padding.spaceX.px,
-    },
-    '&.is_result span': {
-      display: 'none !important',
-    },
-    '&.is_result.first': {
-      paddingTop: 1,
-    },
-  },
+  // eventOptionCollapse: {
```
Comment (dwalintukan) [inline src/scenes/Event/Option/styles.js]: remove if unused

File: src/scenes/Event/store.js [hunk 6]
```diff
@@ -171,14 +173,15 @@ export default class EventStore {
   @action
   initEvent = async () => {
     if (!this.url) return;
-
+    const userAddress = this.app.wallet.currentAddress || null;
     const { items } = await events(this.app.graphqlClient, {
       filter: { OR: [{ txid: this.url }, { address: this.url }] },
       includeRoundBets: true,
+      userAddress,
```
Comment (dwalintukan) [inline src/scenes/Event/store.js]: change once backend name changes

File: src/network/graphql/queries.js [hunk 7]
```diff
@@ -54,6 +53,8 @@ const QUERIES = {
       $skip: Int
       $pendingTxsAddress: String
       $includeRoundBets: Boolean
+      $userAddress: String
```
Comment (dwalintukan) [inline src/network/graphql/queries.js]: change after backend name changes

File: src/network/graphql/queries.js [hunk 8]
```diff
@@ -62,6 +63,8 @@ const QUERIES = {
         skip: $skip
         pendingTxsAddress: $pendingTxsAddress
         includeRoundBets: $includeRoundBets
+        userAddress: $userAddress
```
Comment (dwalintukan) [inline src/network/graphql/queries.js]: change after backend name changes

File: src/scenes/Event/store.js [hunk 9]
```diff
@@ -21,6 +21,7 @@ const { BETTING, ORACLE_RESULT_SETTING, OPEN_RESULT_SETTING, ARBITRATION, WITHDR
 const INIT = {
   loading: true,
   event: undefined,
+  betRoundBets: undefined,
```
Comment (dwalintukan) [inline src/scenes/Event/store.js]: cant you just get this from `event.roundBets`?

Comment (dwalintukan) [reply src/scenes/Event/store.js]: looks like this is unused, delete if unused. when i refactored the event store i tried the store as clean as possible of an `event` duplicated vars. if its already in `event` and we dont need to alter it, then keep it there and access it from `event.`

File: src/scenes/Event/store.js [hunk 10]
```diff
@@ -70,6 +71,7 @@ export default class EventStore {
   @observable totalInvestment = INIT.totalInvestment
   @observable returnRate = INIT.returnRate
   @observable profitOrLoss = INIT.profitOrLoss
+  @observable betRoundBets = INIT.betRoundBets
```
Comment (dwalintukan) [inline src/scenes/Event/store.js]: delete?

File: src/languageProvider/locales/ko-KR.json (hunk 1 above)
Comment (henryzbw) [reply src/languageProvider/locales/ko-KR.json]: lol. I just google translated

**Comment by henryzbw** [comment]
#1328

**Comment by henryzbw** [comment]
#1239

**Comment by henryzbw** [comment]
#1353

**Comment by henryzbw** [comment]
#1354

**Comment by henryzbw** [comment]
#1366

**Pull request merged**